process_report/processors/remove_nonbillables_processor.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+
+import pandas
+
+from process_report.invoices import invoice
+from process_report.processors import processor
+
+
+@dataclass
+class RemoveNonbillablesProcessor(processor.Processor):
```

Comment:
This name is now incorrect. Since there's not a lot happening here, I would suggest moving the logic of this processor inside of the validate billable pi processor.